themes/default/content/blog/pulumi-release-notes-66/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Jan. 2nd releases: Pulumi Packages support for plugins hosted anywhere and Pulumi Service 3rd party audit for secrets decryption"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-01-21T13:20:48-08:00
+
+allow_long_title: true
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: The latest Pulumi updates also include Pulumi import support for Kubernetes CRD, various improvements to Helm Release, and native ES Module support. 
+
```

Comment:
Right now, Hugo is building a `summary` (for display on the index page) dynamically using the `<!--more-->` separator, which is why you see the bullets there:

<img width="778" alt="image" src="https://user-images.githubusercontent.com/274700/150916693-474982fb-f263-4fb8-bcbb-54d76eb7d45a.png">

This is fine, but if you wanted to put something a little more concise and introductory in that space to show up before the Read More link, you could specify a `summary` in the frontmatter params:

```suggestion

summary: A few interesting lines that introduce the post. These will appear here on the blog home page.

```

Up to you, I just figured I'd mention it in case you wanted a little more control over that space.